### DISCUSSION TITLE
meta nfo数据出问题

### DISCUSSION THREAD

**Discussion by ghost** [yutto-dev/yutto#67] [General]
url https://www.bilibili.com/video/BV1Fa41127LF 
命令大约为 fr"""yutto "{BV}" -tp "{BV}" -q 80 --with-metadata -c "xxxx" --no-danmaku
控制台输出
![image](https://user-images.githubusercontent.com/62796978/156823386-1766602c-cee1-45b4-ab08-a43f1bf4b1d9.png)
nfo数据
```
<?xml version="1.0" ?>
<episodedetails>
	<title>1643775239905.mp4</title>
	<show_title>1643775239905.mp4</show_title>
	<plot>-</plot>
	<thumb>http://i0.hdslb.com/bfs/storyff/n220202qn37tzl4ma81uwe27whja74z7_firsti.jpg</thumb>
	<premiered>2022-02-02 12:22:12</premiered>
	<dataadded>2022-03-05 02:36:37</dataadded>
	<source/>
	<original_filename/>
</episodedetails>
```

明显是解析上出现了一些问题

**Comment by dydhyhwu** [comment]
这个提到 `Issues` 里吧，应该是解析的结构有改动

**Comment by SigureMo** [comment]
已转为 issue，后续进展见 #70
